src/index.ts

```diff
@@ -65,4 +74,10 @@ export function getPrivateKey(options: Options = {}): string | null {
   return null;
 }
 
+function addNewlines(privateKey: string): string {
+  const middleLength = privateKey.length - begin.length - end.length - 2;
+  const middle = privateKey.substr(begin.length + 1, middleLength);
+  return `${begin}\n${middle.replace(/\s/g, "\n")}\n${end}`;
```

Comment:
would it make sense to avoid multiple new lines here?
```suggestion
  return `${begin}\n${middle.replace(/\s+/g, "\n")}\n${end}`;
```